outbound/config.js

```diff
@@ -41,6 +42,9 @@ function load_config () {
     if (!cfg.ipv6_enabled && config.get('outbound.ipv6_enabled')) {
         cfg.ipv6_enabled = true;
     }
+    if (!cfg.received_header_disabled && config.get('outbound.received_header_disabled')) {
+        cfg.received_header_disabled = true;
+    }
```

Comment:
This is not necessary. These blocks only exist for backwards compatibility with legacy config files. This is a new setting.